KomaMRIBase/src/sequences/ADC/design.jl

```diff
@@ -0,0 +1,26 @@
+"""
+"""
+function make_adc(num_samles; Δtadc=0, duration=0, delay=0,
```

Comment:
typo num_samles to num_samples

Reply:
This function will be completelly replaced